snuba/api.py

```diff
@@ -242,7 +255,8 @@ def dataset_query(dataset, body, timer):
 
 
 @split_query
-def parse_and_run_query(dataset, validated_body, timer):
+def parse_and_run_query(dataset, query, options, timer):
+    validated_body = {**query, **options}  # XXX: compatibility
```

Comment:
See note in the PR description about this. There are a lot of things downstream of this function that still rely on the query and options being in the same mapping and I don't want to tackle them all at once.